css/filter-effects/inheritance.html

```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
+<html>
+<head>
+<meta charset="utf-8">
+<title>Inheritance of Filter Effects properties</title>
+<link rel="help" href="https://drafts.fxtf.org/filter-effects/#property-index">
+<meta name="assert" content="Properties inherit according to the spec.">
+<meta name="assert" content="Properties have initial values according to the spec.">
+<script src="/resources/testharness.js"></script>
+<script src="/resources/testharnessreport.js"></script>
+<script src="/css/support/inheritance-testcommon.js"></script>
+</head>
+<body>
+<div id="container">
+  <div id="target"></div>
+</div>
+<script>
+assert_inherited('color-interpolation-filters', 'linearrgb', 'srgb');
+assert_not_inherited('filter', 'none', 'blur(10px)');
+assert_not_inherited('flood-color', 'rgb(0, 0, 0)', 'rgb(42, 53, 64)');
```

Comment:
Wha do the 2 functions test?

Reply:
See comments in https://github.com/web-platform-tests/wpt/blob/master/css/support/inheritance-testcommon.js

The first function tests that `color-interpolation-filters` is an inherited property, with `linearrgb` the initial value, and `srgb` being another accepted value (a value other than the initial value is used to test inheritance).

The second function is used to test that `filter` and `flood-color` do not inherit, and have the initial values `none` and `rgb(0, 0, 0)` respectively.

Each function creates 2 tests (initial value and inheritance).